crypto/evp/evp_lib.c

```diff
@@ -266,7 +265,12 @@ int EVP_CIPHER_CTX_encrypting(const EVP_CIPHER_CTX *ctx)
 
 unsigned long EVP_CIPHER_flags(const EVP_CIPHER *cipher)
 {
-    return cipher->flags;
+    unsigned long v = cipher->flags;
+    int ok = evp_do_param(cipher, &v, sizeof(v),
```

Comment:
same comments as with ```EVP_CIPHER_CTX_set_padding```

I won't repeat them any more.

Reply:
Same reasoning for not doing this.